src/server/main.cpp

```diff
@@ -9,7 +9,8 @@ int main(int, char const**) {
     //size_t player_1_id = game_controller.createPlayer();
 
     while (true) {
-        // Update game state based on client inputs
-        // game_controller.update(client_1.getId(), client_1.getDirection());
+        // TODO: What should this amount be set to?
+        sf::sleep(sf::milliseconds(50));
```

Comment:
Not sure what this amount should be. 
If I don't have this here then the server quickly moves the circles (as fast as it can compute) and the client doesn't have time to catch up, so the circle teleports super far away on the first _client_ state update.

This basically determines the games pace -  tick/sec

Reply:
I dont get it, is the server just moving the clients by itself? in the initial design a single tick was equal to the time of accepting move commands (100ms) + time to compute game state. Now it seems artificial and detached from the client actions

Reply:
I'm confused...

So during the accepting move commands 100ms sleep, we should be computing game state?

Reply:
Moved the game state update to `Gamecontroller` in b5490ec